pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/email/SendGridClient.java

```diff
@@ -67,10 +65,11 @@ private void sendEmail(@NonNull String sendGridTemplate, @NonNull Collection<Rec
     /**
      * Sends a request per recipient to SendGrid.
      */
-    private void sendRequestUsingTemplate(@NonNull Recipient recipient, @NonNull String sendGridTemplate, @NonNull SendGrid sendGrid)
+    private void sendRequestUsingTemplate(@NonNull Recipient recipient, @NonNull String sendGridTemplate,
+                                          @NonNull SendGrid sendGrid, JsonObject settings)
```

Comment:
Are we passing an instance variable to a private method (SendGrid)?

Reply:
Not for long! :)